native/cocos/3d/assets/Mesh.cpp

```diff
@@ -629,9 +629,7 @@ bool Mesh::merge(Mesh *mesh, const Mat4 *worldMatrix /* = nullptr */, bool valid
         Uint8Array vbView(vb);
 
         Uint8Array srcVBView = _data.subarray(srcOffset, srcOffset + bundle.view.length);
-        srcOffset += srcVBView.length();
         Uint8Array dstVBView = mesh->_data.subarray(dstOffset, dstOffset + dstBundle.view.length);
-        dstOffset += dstVBView.length();
```

Comment:
Why delete these codes?

Reply:
dstOffset is not used after this line.

Reply:
It's dead code.

Reply:
If so, can delete the variable?

Reply:
Yep, I think we could remove them.

Reply:
I think define a variable is better since it's used more than once.
```c++
Uint8Array srcVBView = _data.subarray(srcOffset, srcOffset + bundle.view.length); // srcOffset is used twice here
Uint8Array dstVBView = mesh->_data.subarray(dstOffset, dstOffset + dstBundle.view.length); // dstOffset is used twice here too
```
If remove them, the code will be more complicated like the following

```c++
Uint8Array srcVBView = _data.subarray(bundle.view.offset, bundle.view.offset + bundle.view.length);
Uint8Array dstVBView = mesh->_data.subarray(dstBundle.view.offset, dstBundle.view.offset + dstBundle.view.length);
```

Reply:
DONE, Remove them in https://github.com/cocos/cocos-engine/pull/12045/commits/8b6a76dc6d802ad12287dc1763a3385a82a542db